### PR TITLE
Document image transformer

### PR DESCRIPTION
**Pull request by DeMoorJasper**
This PR is an attempt at documenting the image transformer

### CODE REVIEW

**Comment by mischnic** [comment]
I think this makes more sense as a "recipe" than a language?

**Comment by DeMoorJasper** [comment]
@mischnic I don't think it belongs in either, not sure but maybe languages should be renamed to `asset types` or something?

**Comment by mischnic** [comment]
Yeah. Generally, I like that languages is more high-level than asset types though.

**Comment by DeMoorJasper** [comment]
@mischnic just checked and in Parcel 1 we actually used to call it asset types, I'm totally fine with adding a new section for `images`, `manifests`, ... just not sure how to call it besides asset types

**Comment by mischnic** [comment]
Can we put in recipes for now? We can always reorganize it once we know what it should be

**Comment by DeMoorJasper** [comment]
@mischnic sure moved it for now,

File: src/recipes/image.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+## Parcel image transformer
+
+Using the Parcel transformer `@parcel/transformer-image` you can resize, change the format and quality of an image. To do this we added the possibility to define query parameters.
+
+The query parameters you can use are:
+
+- `width`: The width to resize the image to
+- `height`: The height to resize the image to
+- `quality`: The image quality you want, for example `75`
```
Comment (mischnic) [inline src/recipes/image.md]:
```suggestion
- `quality`: The image quality percentage you want, for example `75`
```

File: src/recipes/image.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+- `format`: File format to use, for example: `webp`
+
+Supported image formats: `jpeg/jpg`, `png`, `webp`, `tiff`, `heic/heif` and `raw`
```
Comment (mischnic) [inline src/recipes/image.md]:
```suggestion
Supported image formats: `jpeg` / `jpg`, `png`, `webp`, `tiff`, `heic` / `heif` and `raw`
```

I think this is what you mean?

File: src/recipes/image.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+An example:
+
+{% sample %}
+{% samplefile "index.js" %}
+
+```js
+import imageUrl from "url:./image.jpeg?format=webp&width=250";
+```
+
+{% endsamplefile %}
```
Comment (mischnic) [inline src/recipes/image.md:35]: Should we add an example for HTML (`img src=`) as well? Do you need `url:` in that case?

Comment (DeMoorJasper) [reply src/recipes/image.md:35]: Sure, will try it out, I think url is still required although not sure, probably depends on the config?

Comment (DeMoorJasper) [reply src/recipes/image.md:35]: `url:` is required in the default config, so used that for now

**Review by mischnic — APPROVED** [review]
🎉

**Pull request merged**